src/etc/inc/upgrade_config.inc

```diff
@@ -1315,6 +1315,7 @@ function upgrade_047_to_048() {
 			$tempdyn['enable'] = isset($config['dyndns'][0]['enable']);
 			$tempdyn['type'] = $config['dyndns'][0]['type'];
 			$tempdyn['wildcard'] = isset($config['dyndns'][0]['wildcard']);
+			$tempdyn['proxied'] = isset($config['dyndns'][0]['proxied']);
```

Comment:
@rbgarga  Ah.  Sorry about that, I was trying to follow along with the template of others.  We can remove this line.